js/Application.js

```diff
@@ -34,6 +36,23 @@ define(
 				this.densityPriority = 0;
 				this.pageModel = model;
 				this.router = router;
+				this.pageTitle = ko.pureComputed(() => {
+					let pageTitle = "ATLAS";
+					switch (this.router.currentView()) {
+						case 'loading':
+							pageTitle = pageTitle + ": Loading";
+							break;
+						default:
+							pageTitle = `${pageTitle}: ${this.router.activeRoute().title}`;
+							break;
+					}
+
+					if (this.pageModel.hasUnsavedChanges()) {
+						pageTitle = "*" + pageTitle + " (unsaved)";
```

Comment:
Let's be consistent in code style: if you are using templated strings (` `` `), why not to do it everywhere?